format/hint/schema.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "value": {
+      "type": ["string","null"]
```

Comment:
I've made the `value` property optional. 
I handle the `required` constraint in my validator instead.